lang/es.xml

```diff
@@ -223,11 +239,13 @@ No apagues el sistema ni cierres la aplicación.</warning_saving>
 	</settings>
 
 	<trophy_collection>
+		<delete_trophy>Eliminar trofeos</delete_trophy>
+		<trophy_deleted>Se eliminará la información de los trofeos de este usuario.</trophy_deleted>
 		<details>Detalles</details>
 		<earned>Ganado</earned>
 		<name>Nombre</name>
 		<no_trophies>No hay trofeos.
-Podrás conseguir trofeos si usas una aplicación compatible con la función de trofeos.</no_trophies>
+Podrás ganar trofeos si usas una aplicación compatible con la función de trofeos.</no_trophies>
```

Comment:
no replace string exist, is already good

Reply:
I'm sorry, I do not understand that English. If what you mean is that this string was taken from the official translations, please confirm so I can revert this.